### PR TITLE
Feature/errmsg

### PR DESCRIPTION
**Pull request by sureshvarman-cronj**

### CODE REVIEW

File: src/server/i18n/de.json [hunk 1]
```diff
@@ -81,6 +81,8 @@
     "INVALID_RETYPE_PASSWORD": "Kennwörter stimmen nicht überein",
     "PASSWORD_REQUIRED": "Kennwort ist erforderlich",
     "PASSWORD_EXPIRED": "Kennwort ist abgelaufen",
-    "DESCRIPTION": "Unser Entwicklungsteam wird angesprochen über das Thema, wir werden sie so schnell wie möglich reparieren"
+    "DESCRIPTION": "Unser Entwicklungsteam wird angesprochen über das Thema, wir werden sie so schnell wie möglich reparieren",
```
Comment (kwierchris) [inline src/server/i18n/de.json]: Sorry no :D
What is the context of this message?

File: src/server/i18n/de.json [hunk 2]
```diff
@@ -81,6 +81,8 @@
...
+    "404": "! Nicht gefunden",
+		"404_DESCRIPTION": "Die Anfrage-URL wird nicht gefunden"
```
Comment (kwierchris) [inline src/server/i18n/de.json]: "Der Link wurde nicht gefunden"

File: src/server/i18n/de.json (hunk 1 above)
Comment (sureshvarman-cronj) [inline src/server/i18n/de.json]: `Our development team is addressed about the issue, we will fix it as soon as possible`

Comment (kwierchris) [inline src/server/i18n/de.json]: Yeah I know. But this is not enough. Which issue gets addressed?

Comment (sureshvarman-cronj) [inline src/server/i18n/de.json]: There is a log message, but particular message is anyway not going to help the user. Only we need to know, for that we have log.

Comment (kwierchris) [inline src/server/i18n/de.json]: In this case I would also change the english version so we do not make any "promises" like:
EN: "Our development team has been informed about this issue."
- or - "This issue as been reported to our development team."

DE: "Unser Entwicklungsteam wurde über das Problem informiert."
- or - "Der Fehler wurde an unser Entwicklerteam gemeldet."

**Pull request merged**